packages/tsconfig-reference/scripts/cli/generateMarkdown.ts

```diff
@@ -56,8 +56,8 @@ languages.forEach((lang) => {
     <thead>
     <tr>
       <th>Flag</th>
-      <th>Type</th>
-      ${opts?.noDefaults ? "" : "<th>Default</th>"}
+      <th>Type</th>${opts?.noDefaults ? "" : "     <th>Default</th>"}
+      
```

Comment:
Would this newline cause the same problem?

Reply:
No, it's fine.